ts/Core/Series/Point.ts

```diff
@@ -1319,6 +1319,12 @@ class Point {
                     .indexOf(userEvent) === -1
             )
         ) {
+            // While updating the existing callback event the old one should be
+            // removed
+            if (point.hasImportedEvents) {
+                removeEvent(point, eventType);
+            }
```

Comment:
This removes all listeners, while three lines up we are just testing, if the new listener has not been added. It might be better to have somewhere a reference of the outdated listener, to just remove that one and keep the rest of listeners in place. So something like a `splice` operation on `point.hcEvents[eventType]`. But at the same time it reminds of an old discussion about series listeners. Some users like to have the callback replacement while others like to have callback additions.

Reply:
I've changed the solution a little bit. I would say, that callback additions are not the correct way of dealing with it, even in the topic https://github.com/highcharts/highcharts/issues/20435 users complain about old events, which which be updated/removed.